lib/python/ensembl/brc4/runnable/manifest_stats.py

```diff
@@ -168,13 +195,68 @@ def parse_gff3(self, gff3_handle: TextIO) -> List:
             sorted_biotypes[name] = data
         
         stats = [f"{data['unique_count']:>9}\t{biotype:<20}\tID = {data['example']}" for (biotype, data) in sorted_biotypes.items()]
+
+        # Check against NCBI stats
+        stats += self.check_ncbi_stats(biotypes,self.param('accession'))
         
         return stats
+    
+    def check_ncbi_stats(self, biotypes: Dict, accession: str) -> List:
+        """Use the dataset tool from NCBI to get stats and compare with what we have"""
+
+        stats = []
+
+        datasets_bin = self.param('datasets_path')
+        if not which(datasets_bin):
```

Comment:
This "conversion" of `datasets_path` to `datasets_bin` makes the code a little bit confusing: the former gives me the impression it is a path to a folder with dataset files, and the latter that the path must be to a binary that process datasets? Not sure if applicable in this PR, but it would be good to revisit this naming later on.